internal/httpserve/handlers/register.go

```diff
@@ -84,7 +85,11 @@ func (h *Handler) RegisterHandler(ctx echo.Context) error {
 		ID:      meowuser.ID,
 		Email:   meowuser.Email,
 		Message: "Welcome to Openlane!",
-		Token:   meowtoken.Token,
+	}
+
+	// only return the token in development
+	if strings.Contains(ctx.Request().Host, "localhost") {
```

Comment:
This is probably a risky way to do this given you can spoof the host of a request... any reason not to use one of the existing server config flags we have (e.g. `server.dev`) over the request host ?